release/release_gold.sh

```diff
@@ -5,6 +5,40 @@ source _product_info_json.sh
 source _promotion.sh
 source _publishing.sh
 
+RELEASE_PROPERTIES_FILE="release-data/release.properties"
```

Comment:
It is superfluous to create a variable just for this. It is used in a single place only.